content/_es/help/manage-your-account/add-or-change-your-authentication-method.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Añadir o cambiar el método de autenticación
+category: manage-your-account
+order: 6
+---
+[Un método de autenticación](https://login.gov/es/help/get-started/authentication-options/) es una medida adicional de seguridad para tu cuenta. Te aconsejamos que tengas al menos dos métodos de autenticación para tu cuenta en caso de que pierdas uno de ellos.
+
+Si perdiste tu método de autenticación y no tienes una copia de seguridad conectada a la cuenta, [deberás eliminar la cuenta actual de login.gov](https://login.gov/es/help/manage-your-account/delete-your-account/) y crear una nueva.
+
+## Añade o cambia tu método de autenticación:
+
+1. Escribe tu dirección de correo electrónico en <https://secure.login.gov>.
```

Comment:
Can we link to the Spanish version?

```suggestion
1. Escribe tu dirección de correo electrónico en <https://secure.login.gov/es/>.
```

Same applies for French version of this page.